projects/packages/publicize/src/class-publicize-setup.php

```diff
@@ -74,14 +74,21 @@ public static function on_jetpack_feature_publicize_enabled() {
 		add_action( 'rest_api_init', array( static::class, 'register_core_options' ) );
 		add_action( 'admin_init', array( static::class, 'register_core_options' ) );
 
+		if ( ( new Host() )->is_wpcom_simple() ) {
+
+			wpcom_rest_api_v2_load_plugin( Jetpack_Social_Settings\Settings::class );
+		} else {
+			// Load the settings page.
+			new Jetpack_Social_Settings\Settings();
```

Comment:
Don't we have to call register_settings? We called it on the line that was removed

Reply:
Oh I see you added a constructor below!

Reply:
It's handled in its constructor.